doc/docs/sample-replidraw.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Replidraw
+slug: /samples/replidraw
+---
+
+A tiny Figma-like multiplayer graphics editor.
+
+Built with [Replicache](https://replicache.dev), [Next.js](https://nextjs.org/),
+[Fauna](https://fauna.com/), and [React
```

Comment:
Fauna part not correct. Is built with MySQL.

Reply:
Copy paste from Github repo... will update